tests/foreman/cli/test_gpgkey.py

```diff
@@ -79,41 +79,6 @@ def create_gpg_key_file(self, content=None):
 
     # Bug verification
 
-    @skip_if_bug_open('redmine', 4271)
-    def test_redmine_4271(self):
```

Comment:
There is no special check for the CSV output in this test. Also CSV output is being used every time a GPG key is created because the default CLI behaviour. With that said there is no need to this test exist because it is doing a positive create test which already exists.